### ISSUE TITLE
neomake signs not working

### ISSUE THREAD

**Issue by carlitux** [colepeters/base16-spacemacs-redux#4]
When linter finish no signs on left side.

**Comment by colepeters** [comment]
Could this be a problem within your linter config? I’m seeing linting signs just fine over here:

![image](https://user-images.githubusercontent.com/1713932/28441682-f2db0b68-6da3-11e7-8122-29aac79c07de.png)

**Comment by carlitux** [comment]
@colepeters there is any requirement? I am using now monokai and signs are working.

**Comment by colepeters** [comment]
@carlitux No, there are no requirements. Which plugin are you using for linting, and what is your configuration?

**Comment by carlitux** [comment]
https://github.com/neomake/neomake on javascript with eslint and flow

**Comment by colepeters** [comment]
Can you also paste your neomake config?

**Comment by carlitux** [comment]
Just this

```vim
let g:neomake_javascript_enabled_makers = ['eslint', 'flow']
let g:neomake_autolint_sign_column_always = 1
let g:neomake_ft_maker_remove_invalid_entries = 1
```
